docs/build/configure/_index.md

```diff
@@ -264,6 +264,123 @@ The corresponding raw JSON looks like this:
 
 {{% /expand %}}
 
+## Webhooks
+
+Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud.
+For example, you can configure a webhook to send you a phone notification when your robot's battery drops below 10%.
+
+There is no webhooks tab in the Viam app config builder; use **Raw JSON** mode to configure webhooks as follows:
```

Comment:
Hmm, torn here. I might go with:

```suggestion
You can configure webhooks in the Viam app, using **Raw JSON** mode:
```

And then add a step 1 below:

> 1. Navigate to the [Viam app](https://app.viam.com/robots) and switch to **Raw JSON** mode.

But you're right that users could well (are very likely to) be expecting a UI presence in Builder mode, like so many of our other configurables. OK by me to keep "not where you expect it", but instead of "tab", what do you think of:

```suggestion
You can configure webhooks in the Viam app, using **Raw JSON** mode.
Webhooks do not support  **Builder** mode configuration at this time:
```

Up to you, happy either way!